### PR TITLE
Suppress trace output.

### PR DESCRIPTION
**Pull request by pkofod**

### CODE REVIEW

File: test/multivariate/solvers/zeroth_order/particle_swarm.jl [hunk 1]
```diff
@@ -27,11 +27,12 @@
     res = Optim.optimize(rosenbrock_s, initial_x, ParticleSwarm(lower, upper, n_particles),
                              options)
     @test norm(Optim.minimizer(res) - [1.0, 1.0]) < 0.1
-
+    @suppress_out begin
     # TODO: Is there a way to suppress the trace results?
```
Comment (anriseth) [inline test/multivariate/solvers/zeroth_order/particle_swarm.jl]: You can remove this line now :D

File: test/REQUIRE [hunk 2]
```diff
@@ -1,2 +1,3 @@
 OptimTestProblems 1.2
 RecursiveArrayTools
+Suppressor
```
Comment (anriseth) [inline test/REQUIRE]: Seems unhappy with your newline?

Comment (pkofod) [reply test/REQUIRE]: what's unhappy?

Comment (pkofod) [reply test/REQUIRE]: ah, it didn't show out here in the comments

**Pull request merged**